diagnostic_aggregator/CMakeLists.txt

```diff
@@ -129,6 +130,27 @@ if(BUILD_TESTING)
     )
   endforeach()
 
+  set(add_analyzers_tests
+  "all_analyzers")
+
+  foreach(test_name ${add_analyzers_tests})
```

Comment:
I followed the previous pattern even though I only provide a test for one of them. I can remove the loop or add the expected outcome for the other analyzers, or just leave this as is.